src/__tests__/cucumber_tests/shared_steps/equalizerApo.ts

```diff
@@ -36,6 +36,7 @@ export const givenCanWriteToAquaConfig = (given: DefineStepFunction) => {
       isEnabled: false,
       isAutoPreAmpOn: false,
       isGraphViewOn: false,
+      isCaseSensitiveFs: false,
```

Comment:
Scary thing for this to happen. I wonder if theres some way to prevent an object to be created without some missing property.

Reply:
That's the risk with JS/TS unfortunately 🙃